BaseTools/Plugin/OverrideValidation/OverrideValidation.py

```diff
@@ -582,6 +600,14 @@ def path_parse():
         '-r', '--regenpath', dest = 'RegenPath', type=str,
         help = '''Specify the absolute path to an inf with existing overrides to regen by passing r Path/To/Target or --regenpath Path/To/Target.'''
         )
+    group.add_argument (
+        '-d', '--deprecatepath', dest = 'DeprecatedPath', type=str,

```

Comment:
Would you need to set default to None given you are comparing it against None to see if this is set?